solutions/cf-1365D.mdx

```diff
@@ -40,97 +40,103 @@ typedef long long ll;
 const int mxN = 55;
 char grid[mxN][mxN];
 bool visited[mxN][mxN];
+int rowMovement[4] = {0, 1, 0, -1};
+int columnMovement[4] = {1, 0, -1, 0};
+
 int N, M;
 bool can_go = 0;
```

Comment:
not sure why you only converted one boolean to true / false. What about lines 47, 75, 108?

Reply:
i think there was something messed up because my java solution was merged. there is no way that 3 reviewers would have missed this. I'll fix everything that got messed up